LuaParser/src/Parse/LuaParser.cpp

```diff
@@ -218,6 +218,8 @@ void LuaParser::IfStatement() {
     }
     CheckAndNext(TK_END);
 
+    TestAndNext(';');
```

Comment:
Do we really need to consider 'if a then d end;'? In C language, it is generally not common to include a semicolon after the 'if' statement.

Reply:
The codebase I work on (given, I mostly need the stripping of semicolons for that) contains a lot of semicolons after if statements and similar. And currently, they are then seen as empty statements which are (with default configuration) put on a new line, but keeping it with the statement seems more natural.
Maybe there'd need to be an additional "C-Style" configuration differentiating between statements where to put statements and where not. Or the "always" configuration needs to be adapted to that. But with the current implementation of "always", it also needs to be considered as the implementation puts a semicolon if there isn't already one in the statement-node.

Reply:
I accept